Fabric.Authorization.API/Modules/IdentitySearchModule.cs

```diff
@@ -50,11 +54,7 @@ public class IdentitySearchModule : SearchModule<IdentitySearchRequest>
             catch (NotFoundException<Role> ex)
             {
                 return CreateFailureResponse(ex.Message, HttpStatusCode.NotFound);
-            }
-            catch (Exception ex)
```

Comment:
Was catching this general exception causing us to inadvertently swallow something?

Reply:
Yes, this was causing the model validation error to be swallowed so the client did not know the cause of the error.

Reply:
I'm trying to remember why I was catching the general Exception here. Not sure if it had to do with not being able to reach Fabric.Identity or something going wrong with that call out or if it was related to the Content-Type being set on the GET request.